### ISSUE TITLE
[HOLD #35539][#VIP-VSB] Thread in #admin shows Hidden with avatar placeholder in LHN for workspace member

### ISSUE THREAD

**Issue by kavimuru** [Expensify/App#35132]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___
WAVE/PROJECT SUGGESTION : #vip-vsb

**Version Number:** v1.4.31-7
**Reproducible in staging?:** y
**Reproducible in production?:** y
**If this was caught during regression testing, add the test name, ID and link from TestRail:**
**Email or phone of affected tester (no customers):**
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Expensify/Expensify Issue URL:**
**Issue reported by:** Applause internal team
**Slack conversation:**

## Action Performed:
Precondition:
- User A is an admin of Workspace A.
- User B is a member of Workspace A.

1. [User B] Create a room under Workspace A.
2. [User B] Invite User C to the room.
3. [User A] Go to #admin room of Workspace A.
4. [User A] Reply in thread to the system message "User B invited User C to room".
5.[User B] Note the thread in LHN.

## Expected Result:
The thread in LHN will appear with thread title and complete avatar for User B.

## Actual Result:
The thread in LHN appears with "Hidden" as thread title and avatar placeholder for User B. The thread in LHN only shows the title and avatar after user opens the thread. On production, it is the same behavior except that it shows "Deleted message".

## Workaround:
Can the user still use Expensify without this being fixed? Have you informed them of the workaround?

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android: Native
- [ ] Android: mWeb Chrome
- [ ] iOS: Native
- [ ] iOS: mWeb Safari
- [x] MacOS: Chrome / Safari
- [ ] MacOS: Desktop

## Screenshots/Videos

Add any screenshot/video evidence

https://github.com/Expensify/App/assets/43996225/e453bdba-958e-456e-ac5d-5f2b34ac5063

</details>

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

<details><summary>Upwork Automation - Do Not Edit</summary>
    <ul>
        <li>Upwork Job URL: https://www.upwork.com/jobs/~01403dd68ad2cbea53</li>
        <li>Upwork Job ID: 1750411743673921536</li>
        <li>Last Price Increase: 2024-02-01</li>
    </ul>
</details>

**Comment by Santhosh-Sellavel** [comment]
I'm unavailable next week, Please assign a new C+ Issue here if required, thanks!

cc: @CortneyOfstad

**Comment by CortneyOfstad** [comment]
Sounds good @Santhosh-Sellavel — we should be good, but thanks for the heads up! 👍

**Comment by CortneyOfstad** [comment]
@Santhosh-Sellavel thoughts on the proposal above? Thanks!

**Comment by CortneyOfstad** [comment]
@Santhosh-Sellavel does the proposal above work, or does it need feedback? Thanks!

**Comment by Santhosh-Sellavel** [comment]
Will Review today

**Comment by Santhosh-Sellavel** [comment]
@CortneyOfstad This seems like backend issue can you add internal

**Comment by Santhosh-Sellavel** [comment]
> I suggest we show a different title for the thread. This is because we don't have the correct data. I'm suggesting to modify the logic added by this pr #34362.

Can you elaborate on that @nebiyuelias1

**Comment by Santhosh-Sellavel** [comment]
> @CortneyOfstad This seems like backend issue can you add internal

bump @CortneyOfstad

**Comment by CortneyOfstad** [comment]
Not overdue — waiting for an engineer to pick it up 👍

**Comment by CortneyOfstad** [comment]
Still waiting on an engineer for this one 👍

**Comment by CortneyOfstad** [comment]
Reached out in the room since there is not an urgency applied. Waiting to hear back to get this moving 👍

**Comment by CortneyOfstad** [comment]
Bumped the thread in Slack [here](https://expensify.enterprise.slack.com/archives/C066HJM2CAZ/p1708112336912189) to get an urgency applied. If that doesn't move an engineer to tackle this.

**Comment by CortneyOfstad** [comment]
Alright, so after checking with the team, this will be corrected by https://github.com/Expensify/App/issues/35539. Adjusting the frequency to reflect that 👍

**Comment by CortneyOfstad** [comment]
No update as there was not any movement on the on-hold issue. Going to keep an eye on both to see when a PR is created.

**Comment by CortneyOfstad** [comment]
No movement yet on the GH https://github.com/Expensify/App/issues/35539 yet. Should be pretty quick this week, as it is shown as "high" priority. Will continue to monitor.

**Comment by CortneyOfstad** [comment]
No movement yet on the other GH, but Puneet linked another issue 24 minutes ago 👍

**Comment by CortneyOfstad** [comment]
Hey @mallenexpensify! I am heading OoO and will be back March 11th, so reassigned to have someone keep this moving until I am back. At this stage, we're on-hold for https://github.com/Expensify/App/issues/35539. It shows that Puneet has requested a query in relation to that other GH. Thanks!

**Comment by mvtglobally** [comment]
Issue not reproducible during KI retests. (First week)

**Comment by mallenexpensify** [comment]
No action needed here so far, we're still on hold pending
- https://github.com/Expensify/App/issues/35539 

The PR for that is actively being worked on by Puneet, he commented 13 mins ago
- https://github.com/Expensify/Expensify/issues/373736

**Comment by CortneyOfstad** [comment]
Back from OoO — thanks for holding down the fort @mallenexpensify!

No action needed, as still waiting for the on-hold issue 👍

**Comment by mallenexpensify** [comment]
- https://github.com/Expensify/App/issues/35539#issuecomment-1980993855

Waiting on query.

**Comment by mvtglobally** [comment]
Issue not reproducible during KI retests. (Second week)

**Comment by mvtglobally** [comment]
Issue not reproducible during KI retests. (Third week)

**Comment by CortneyOfstad** [comment]
I'm unable to recreate this either, so going to close 👍 

Thanks @mvtglobally for continuing to test this!
